matchms/filtering/require_precursor_mz.py

```diff
@@ -0,0 +1,34 @@
+from ..typing import SpectrumType
+from ..typing import Union
```

Comment:
```suggestion
from typing import Union
from ..typing import SpectrumType
```

Reply:
bit confusing, but one is the matchms.typing and one is the general typing module

Reply:
Yes of course, well spotted :)